PoGo.NecroBot.CLI/Settings.cs

```diff
@@ -100,7 +100,6 @@ public static GlobalSettings Load(string path)
             {
                 settings = new GlobalSettings();
             }
-            Logger.Write($"nigger {fullPath} {ConfigPath} {ProfilePath}");
```

Comment:
That was a debug output. Sorry for the n word, not racist ;) I've been using this for ages to mark my debug outputs